crate_universe/src/rendering.rs

```diff
@@ -164,6 +164,14 @@ impl Renderer {
                     tags: BTreeSet::from(["manual".to_owned()]),
                 });
             }
+
+            for extra_aliased_target in &krate.extra_aliased_targets {
+                dependencies.push(Alias {
```

Comment:
We should check this `dependencies` `Vec` for duplicates after we've pushed these all in, to make sure there are no duplicates, and give a descriptive error message if we find any duplicates.

Before they happened to be de-duplicated by construction, but now that we're taking in user-input, we don't have that guarantee any more.